### ISSUE TITLE
`mktemp: (null)` prevents startup of stock 10.8 images

### ISSUE THREAD

**Issue by radarsymphony** [tiredofit/docker-mariadb#22]
## Summary

Recently, I was testing updating some services that use the tiredofit/mariadb:10.8 image. Pulling fresh copy of that image prevented the stacks from restarting correctly. The docker-compose logs show: 

```
example-db  | Image:  tiredofit/mariadb:10.8
example-db  | Repository/Issues/Support:  https://github.com/tiredofit/docker-mariadb/
example-db  |            Sponsor me for development and upkeep: https://www.tiredofit.ca/sponsor
example-db  | 
example-db  | 2023-02-07.22:09:37 [NOTICE] ** [monitoring] Container configured for monitoring with 'zabbix classic'
example-db  | 2023-02-07.22:09:37 [NOTICE] ** [scheduling] Container configured for scheduled tasks with 'cron'
example-db  | mktemp: (null): No such file or directory
example-db  | cp: can't create '': No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | /assets/functions/00-container: line 1241: : No such file or directory
example-db  | mktemp: (null): No such file or directory
example-db  | cp: can't create '': No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | sed: : No such file or directory
example-db  | /assets/functions/00-container: line 1241: : No such file or directory
example-db  | 2023-02-07.22:09:37 [WARN] ** [mariadb] MariaDB data directory is not found, creating initial DB(s)
example-db  | mktemp: (null): No such file or directory
example-db  | /etc/cont-init.d/10-mariadb: line 164: : No such file or directory
example-db  | 2023-02-07.22:09:37 [NOTICE] ** [mariadb] Creating database database
example-db  | /etc/cont-init.d/10-mariadb: line 177: : No such file or directory
example-db  | /etc/cont-init.d/10-mariadb: line 186: : No such file or directory
example-db  | chown: : No such file or directory
example-db  | /etc/cont-init.d/10-mariadb: line 199: : No such file or directory
example-db  | 2023-02-07.22:09:37 [ERROR] ** [mariadb-backup] MariaDB has not properly initialized!
example-db  | 2023-02-07.22:09:37 [STARTING] ** [mariadb] [1] Starting MariaDB 10.8.7 
example-db  | 2023-02-07.22:09:37 [STARTING] ** [monitoring] [1] Starting Zabbix Agent (classic) 6.2.6
example-db  | 2023-02-07.22:09:37 [STARTING] ** [scheduling] [1] Starting cron
example-db  | 2023-02-07.22:09:38 [STARTING] ** [mariadb] [2] Starting MariaDB 10.8.7 
```
The stack db then enters into run-away and the service is eventually stopped.

<details><summary>docker-compose.yml</summary>

```yml
version: '2.4'
services:

  example-db:
    image: tiredofit/mariadb:10.8
    container_name: example-db
    volumes:
      - ./db:/var/lib/mysql
      - ./dbbackup:/backup
      - ./logs:/logs
    environment:
      - DB_NAME=database
      - DB_USER=user
      - DB_PASS=****
      - ROOT_PASS=*****
      - DB_BACKUP=TRUE
      - DB_BACKUP_DUMP_FREQ=1440
      - DB_BACKUP_DUMP_BEGIN=0000
      - DB_BACKUP_CLEANUP_TIME=8640
      - DB_BACKUP_MD5=TRUE
      - DB_BACKUP_COMPRESSION=XZ
      - DB_BACKUP_SPLIT_DB=FALSE
      #- DEBUG_MODE=TRUE
    restart: always

```

</details>

I cloned your mariadb repo and began experimenting building your mariadb image with different base alpine images. The issue seems to have started with the alpine `7.7.41` release, perhaps with [one of these changes](https://github.com/tiredofit/docker-alpine/compare/7.7.40...7.7.41). 

The error `example-db  | 2023-02-07.22:09:37 [ERROR] ** [mariadb-backup] MariaDB has not properly initialized!` appeared throughout all of the images I tested, but did not seem to prevent container startup, so I haven't investigated further yet.

From the docker-compose logs, it looks like the `mktemp` command in the [`update_template` function](https://github.com/tiredofit/docker-alpine/blob/main/install/assets/functions/00-container#L1229) is not being interpreted correctly. I have been unable to narrow it down further yet, but I will update this issue if I can.

_I wasn't sure whether to log this in the Mariadb or the Alpine repo. I've only noticed the issue arising with mariadb so far, so I chose here._

**Comment by tiredofit** [comment]
Neat! Wondered when that one would come back to haunt me.
Fixed with https://github.com/tiredofit/docker-mariadb/commit/b34f6eca64b52254951c0cdff343db44e7888c7c.

The other contstant Mariadb errors that you have on startup are related to mariadb-backup functionality as stated.

**Comment by radarsymphony** [comment]
Thank you. I can confirm that this has resolved the issue for me in `:10.8`
